app/src/main/java/com/devshish/internship/presentation/ui/MainActivity.kt

```diff
@@ -42,20 +44,67 @@ class MainActivity : AppCompatActivity() {
             )
         )
 
-        val navHostFragment = supportFragmentManager
-            .findFragmentById(R.id.navHostFragment) as NavHostFragment
-        navController = navHostFragment.findNavController()
+        with(binding) {
+            ivToggle.setOnClickListener { viewModel.toggle() }
 
-        binding.bottomNavView.setupWithNavController(navController)
+            clPlayerBar.setOnClickListener { viewModel.onPlayerClick() }
 
-        navController.addOnDestinationChangedListener { _, destination, _ ->
-            when (destination.id) {
-                R.id.splashFragment,
-                R.id.authFragment,
-                R.id.webFragment,
-                R.id.playerFragment -> binding.bottomNavView.isVisible = false
-                else -> binding.bottomNavView.isVisible = true
+            val navHostFragment = supportFragmentManager
+                .findFragmentById(R.id.navHostFragment) as NavHostFragment
+            navController = navHostFragment.findNavController()
+
+            binding.bottomNavView.setupWithNavController(navController)
+
+            navController.addOnDestinationChangedListener { _, destination, _ ->
+                when (destination.id) {
+                    R.id.splashFragment,
+                    R.id.authFragment,
+                    R.id.webFragment,
+                    R.id.playerFragment -> binding.bottomNavView.isVisible = true
+                    else -> binding.bottomNavView.isVisible = true
+                }
```

Comment:
1. double true here 
2. You can do something like this:

![image](https://user-images.githubusercontent.com/23121846/140091864-4d96efea-e41f-4b37-96e1-6744da1018dc.png)